README.md

```diff
@@ -1,2 +1,24 @@
-Repository for the Enterprise Certificate Proxy project. This will host Golang source code for keystore daemons for interacting with OS-specific keystores (ex. Keychain on MacOS), which will be distributed as binaries via gCloud SDK. May also contain utils for facilitating TLS offloading and working with OpenSSL.
+# Google Proxies for Enterprise Certificates
 
+Google Enterprise Certificate Proxies are part of the [Google Cloud Zero Trust architecture][zerotrust] that enables mutual authentication with [client-side certificates][clientcert]. This repository contains a set of proxies/modules that can be used by clients or toolings to interact with certificates that are stored in [protected key storage systems][keystore].
+
+To interact the client certificates, application code should rarely need to use most of these proxies within this repository directly, but the clients and toolings provided by Google such as [Cloud SDK](https://cloud.google.com/sdk) to have a more convenient developer experiences to callers.
```

Comment:
"should not need to use the proxies in this repository directly. Instead, use the clients..."